bin/node/runtime/src/lib.rs

```diff
@@ -495,17 +496,9 @@ impl pallet_staking::Config for Runtime {
 		pallet_collective::EnsureProportionAtLeast<_3, _4, AccountId, CouncilCollective>
 	>;
 	type SessionInterface = Self;
-	type RewardCurve = RewardCurve;
+	type EraPayout = pallet_staking::ConvertCurve<RewardCurve>;
```

Comment:
We need to check payouts

Reply:
Let's do it together

Reply:
Tested. Validators is able to get their payouts.